crates/ruff_server/docs/setup/HELIX.md

```diff
@@ -0,0 +1,58 @@
+## Helix Setup Guide for `ruff server`
+
+First, open the language configuration file for Helix. On Linux and macOS, this will be at `~/.config/helix/languages.toml`,
+and on Windows this will be at `%AppData%\helix\languages.toml`.
+
+Add the language server by adding:
+
+```toml
+[language-server.ruff]
+command = "ruff"
+args = ["server", "--preview"]
+```
+
+Then, you'll register the language server as the one to use with Python.
+If you don't already have a language server registered to use with Python, add this to `languages.toml`:
+
+```toml
+[[language]]
+name = "python"
+language-servers = ["ruff"]
+```
+
+Otherwise, if you already have `language-servers` defined, you can simply add `"ruff"` to the list. For example,
+if you already have `pylsp` as a language server, you can modify the language entry as follows:
```

Comment:
We could also provide an example configuration to disable hover (similar to Neovim) if Helix has the same problem as Neovim which is that the client will ask for hover to all the configured servers but Ruff only provides for `# noqa` comments. So, Neovim complains that there's no information via Ruff.

<img width="864" alt="Screenshot 2024-04-29 at 22 09 43" src="https://github.com/astral-sh/ruff/assets/67177269/fc5e8f7b-3833-4762-9b20-c11726323d93">

You can confirm if it's the same problem in Helix and if it is, we can recommend a similar config:

```toml
[[language]]
name = "python"
language-servers = [ { name = "ruff", except-features = [ "hover" ] }, "pyright" ]
```

Reply:
Hover information is somewhat hidden by default in Helix (you have to press `Space-k` to access it, and it opens a new popup window). So I think we're good as-is.

Reply:
Yes, it's similar in Neovim. It's fine to not add it, I was just wondering if Helix also shows a log message like the one in Neovim's case as shown above with "No information available".